2-mandatory/1-create-functions.js

```diff
@@ -33,7 +36,9 @@ Write a function that:
 - Returns a new array containing the same elements, but without the element at the passed index.
 */
 
-function remove() {
+function remove(arr, index) {
+  const newArr = arr.filter((v, i) => index == i);
```

Comment:
Similar comment as before, we should not use filter if we know which element we don't need:
```
const removedElement = arr.splice(index,1)
return arr // this will not contain removedElement
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice